examples/https-redirect/variables.tf

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "region" {
+  default = "us-east1"
+}
+
+variable "zone" {
+  default = "us-east1-b"
+}
+
+variable "network_name" {
+  default = "tf-lb-https-redirect-nat"
+}
+
+variable "service_account" {
```

Comment:
This is probably overly complicated for an example. I'd recommend we just create the service account within the example (using Terraform).

Reply:
I would say leave it as it is. The other examples have it like that. I copied those.

Reply:
The other examples are overly complex in that way, so we shouldn't simply be copying bad habits over. (Though if you want to go back and fix all the examples that's certainly appreciated.)

Our gold standard is that examples should ~only require a project_id.

Reply:
I fixed them all :D